### PR TITLE
Disabling textarea resizing

### PR DESCRIPTION
**Pull request by dominiquemb**

### CODE REVIEW

File: codalab/apps/web/static/less/main.less [hunk 1]
```diff
@@ -284,3 +284,7 @@ This code is for 768px and up
     white-space: pre-wrap;
   }
 }
+
+body textarea {
+	resize: none;
```
Comment (ckcollab) [inline codalab/apps/web/static/less/main.less]: Annoying nitpick: fix indentation?

Can we allow vertical resizing?

**Comment by dominiquemb** [comment]
Vertical resizing enabled, indentation fixed

**Comment by ckcollab** [comment]
Awesome, @dragonlocke will take a look at this on your server when he gets a chance, then after he OK's it we'll merge -- thanks!

**Comment by dominiquemb** [comment]
@dragonlocke Send me a ping on slack when you're ready to test this so that I can switch to the right branch. Thanks!

**Comment by dragonlocke** [comment]
@dominiquemb That's a greenlight, great work!

**Comment by ckcollab** [comment]
:shipit:

**Pull request merged**